packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/ButtonWidget.tsx

```diff
@@ -25,18 +25,13 @@ const useStyles = makeStyles<Theme, ButtonStyleProps>((theme) => ({
     backgroundColor: ({ backgroundColor }) =>
       backgroundColor ? getCSSColor(backgroundColor, theme) : theme.palette.primary.light,
     color: ({ foregroundColor }) => (foregroundColor ? getCSSColor(foregroundColor, theme) : 'white'),
-    fontSize: ({ fontSize }) => (fontSize ? fontSize : null),
-    fontStyle: ({ italic }) => (italic ? 'italic' : null),
-    fontWeight: ({ bold }) => (bold ? 'bold' : null),
+    fontSize: ({ fontSize }) => (fontSize ? fontSize : undefined),
```

Comment:
Are you sure about that? In https://github.com/eclipse-sirius/sirius-web/issues/2144, we changed it for `null` specifically.
(same for all other properties)